specs/specTemplate/sampleComponentSpec.md

```diff
@@ -109,14 +167,31 @@ Detailed description of how input behaves. Logic, corner cases, patterns, etc.
 
 
 
-### Touch
+#### **Touch**
 
 | Event | Action              | NOTE                    |
 | ----- | ------------------- | ----------------------- |
 | tap   | what happens on tap | Side note (if relevant) |
 | drag  |                     |                         |
 
 
+
+## RTL
```

Comment:
we don't really have general guidelines yet :)
also, because this serves as a template, it should have all sections, and you'll get to remove the ones that are redundant to you.